index.js

```diff
@@ -142,7 +209,21 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+  let highest = movies[0];
+  let boxOff = movies[0];
+  if (!movies.length){
+    return null;
+  }
+  for (let i = 1; i < movies.length; i++){
+    if (highest.boxOffice < movies[i].boxOffice)
+        highest = movies[i].boxOffice;
+        boxOff = movies[i];
+      }
+      return highest.title;
```

Comment:
So close! Take a look at what we are setting highest to and then ask yourself - does this value have a title attribute